src/Avalonia.Visuals/Media/FontFamily.cs

```diff
@@ -12,7 +10,7 @@ public sealed class FontFamily
     {
         static FontFamily()
         {
-            Default = new FontFamily(FontManager.Default.DefaultFontFamilyName);
+            Default = new FontFamily("Default");
```

Comment:
Should this maybe use a more unique string than `Default`? What if there's a font called `Default`? Maybe something like `$Default` off the top of my head?

Reply:
Y lets do that. `$Default` is more unique.